docs/introduction/intro/quickstart-entities.mdx

```diff
@@ -40,14 +40,14 @@ Follow these 5 steps to connect your user to Google Calendar and create an event
 </Step>
 
 <Step title="Authenticate Google Calendar Account">
-We'll use Jessica as our example user. There are multiple ways to authenticate an account:
+We'll use **`default`** as our example user. There are multiple ways to authenticate an account:
```

Comment:
Consider adding a brief note explaining that "default" is just an example entity ID and users should replace it with their own entity identifier in production code. This would help prevent confusion.